code-challenges.test.js

```diff
@@ -10,50 +10,150 @@
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers 
+// and returns an array with all the numbers multiplied by 3.
 
+
+// Psuedo Code: Create a function called multiplyByThree
+        //      Parameters: 1 [array]
+        //      Iterate over array using .map method; mulltiply each value by three
+        //      array.map(value => value * 3 )
+        //      Multiply each value by 3
+        //      return: Mapped array
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
+//  describe("multiplyByThree", () => {
+//    it("returns an array with all the numbers multiplied by 3", () => {
+//          expect(multiplyByThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+//          expect(multiplyByThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+//        })
+//  })
+
+ // b) Create the function that makes the test pass.
+// const multiplyByThree = (array) => {
+//    return array.map((value) => value * 3)
+//}
+
+// const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+// const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
-// b) Create the function that makes the test pass.
+// Ran Test
+// ReferenceError: multiplyByThree is not defined
+
+// Ran yarn jest--> Test Suites: 1 failed, 1 total
+//                  Tests:       1 failed, 1 total
+
+// Ran yarn jest--> Test Suites: 1 passed, 1 total
+//                  Tests:       1 passed, 1 total
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+// --------------------1) Create a function that takes a object as an argument and 
+//                        decides if the number inside it is evenly divisible by three or not.
 
+// Prompt: Create a function which takes a object.--> const evedivThree(object). 
+// Inputs: Object
+// (Note: Instructor node...)-->emphasis...
+// Outputs: numbers inside it is easily divisible by three or not. 
+// 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
+// describe("evedivThree", () => {
+//   it("decides if the number inside it is evenly divisible by three or not.", () => {
+//     expect(evedivThree()).toEqual("15 is divisible by three")
+//    // expect(evedivThree(obj)).toEqual("15 is divisible by three")
+//   //  expect(evedivThree(obj)).toEqual("15 is divisible by three")
+//   })
+// })
+
+//     ReferenceError: evedivThree is not defined
+//     TypeError: Cannot read properties of undefined (reading 'values')
+// const evedivThree = (object) => object.keys.values.map((values) =>`${value} % 3 === 0) is divisible by three`)
```

Comment:
I don't see any pseudocode for this problem.  Make sure to provide your process on how you solved this problem and whether or not your test passed.